docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,108 +1,75 @@
 
-experimental[]
+To get started, refer to
+{ingest-guide}/ingest-management-getting-started.html[Quick start: Get logs and metrics into the Elastic Stack].
 
-To visualize data in a CSV, JSON, or log file, you can upload it using the File 
-Data Visualizer. On the home page, click *Upload a file*, and 
-then drag your file onto the *File Data Visualizer*. Alternatively, you can open 
-it by navigating to *Machine Learning* from the side navigation and selecting 
+[role="screenshot"]
+image::images/add-data-fleet.png[Add data using Fleet]
 
-*Data Visualizer*.
+NOTE: This feature is not intended for use as part of a
+repeated production process, but rather for the initial exploration of your data.
 
-[role="screenshot"]
-image::images/ingest-data.png[File Data Visualizer on the home page]
+[discrete]
+[[upload-data-kibana]]
+=== Upload a file
 
-You can upload a file up to 100 MB. This value is configurable up to 1 GB in
+experimental[] If your data is in a CSV, JSON, or log file, you can upload it using the File
+Data Visualizer. You can upload a file up to 100 MB. This value is configurable up to 1 GB in
 <<kibana-ml-settings, Advanced Settings>>.
 
 [role="screenshot"]
 image::images/add-data-fv.png[File Data Visualizer]
 
-The File Data Visualizer uses the {ref}/ml-find-file-structure.html[find_file_structure API] to analyze
-the uploaded file and to suggest ingest pipelines and mappings for your data.
+For geospatial data, you can use the File Data Visualizer or the
+GeoJSon Uploader, depending on the format of your data.  See
+<<import-geospatial-data, Import geospatial data>> for details.
 
 NOTE: This feature is not intended for use as part of a
 repeated production process, but rather for the initial exploration of your data.
 
-[float]
-[[upload-geoipdata-kibana]]
-=== Upload geospatial data
-
-To visualize geospatial data in a point or shape file, you can upload it using the <<import-geospatial-data, GeoJSON Upload>>
-feature in Maps, and then use that data as a layer in a map.
-The data is also available for use in the broader Kibana ecosystem, for example,
-in visualizations and Canvas workpads.
-With GeoJSON Upload, you can upload a file up to 50 MB.
-
-[float]
-[[add-data-tutorial-kibana]]
-===  Index metrics, log, security, and application data
 
-The built-in data tutorials can help you quickly get up and running with
-metrics data, log analytics, security events, and application data.
-These tutorials walk you through installing and configuring a
-Beats data shipper to periodically collect and send data to {es}.
-You can then use the pre-built dashboards to explore and analyze the data.
+[discrete]
+=== Custom options for loading your data
 
-You access the tutorials from the home page.
-If a tutorial doesn’t exist for your data, go to the {beats-ref}/beats-reference.html[Beats overview]
-to learn about other data shippers in the Beats family.
+If the built-in options don't work for you, you can index your
```

Comment:
Similar to my comment above, thoughts on replacing "If the built-in options don't work for you" with "If one of our multiple ingest options don't work for you"?